main.scss

```diff
@@ -13,6 +13,8 @@
 	@import "src/scss/print";
 	@import "src/scss/search";
 	@import "src/scss/tools";
+	@import "src/scss/themes/dark-on-light";
```

Comment:
Cool, I'll change that, add transparent, fix a couple bugs I noticed and then look into merging